src/smart-components/group/add-group/schema.js

```diff
@@ -67,6 +67,9 @@ export default {
         {
           name: 'review',
           title: 'Review details',
+          // eslint-disable-next-line react/display-name
+          buttons: (props) => <ReviewStepButtons {...props} context={AddGroupWizardContext} />,
```

Comment:
Is there any specific reason why you can't use `useContext` inside the `AddGroupWizardContext` component?

Reply:
@Hyperkid123 The ReviewStepButtons component is the same for the add-group wizard as for the add-role wizard. So in the ReviewStepButtons I need to pass a correct context to the useContext hook. Is there any other way to achieve the same thing? I may be using it wrong.

Reply:
Ok. We can leave it for now. We can think of some nice way of switching the context later.